src/assets/lang/es.json

```diff
@@ -54,7 +54,7 @@
     "updates": "Datos de COVID en NY",
     "symptomCheck": "Mi Registro de Salud",
     "contactTracing": "Mis Alertas de COVID",
-    "settings": "Configuración\nde la aplicación"
+    "settings": "Config.\nde la aplicación"
```

Comment:
I think this is a wrong approach, but that's another discussion,

Reply:
Latest commit removes the `\n`